docs/contacts/activity/activity.md

```diff
@@ -2,8 +2,8 @@
 
 Activities are interactions that a team member has had with a contact and are recorded on that record’s timeline. Activities include emails, calls, tasks, notes and meetings. Activities also have the ability to be filtered so that team members can view last communication, ticket replies, or a variety of other helpful updates.
 
-- [Training: How to Log an email](/docs/contacts/activity/logging-emails.md)
```

Comment:
The steps within the logging email trainings aren't numbered, not sure that's a huge deal but wanted to point that out

Reply:
Yeah not a huge deal, but I think it should be fixed.